src/python/gudhi/wasserstein/quantization.py

```diff
@@ -0,0 +1,218 @@
+# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+# Author(s):       Theo Lacombe
+#
+# Copyright (C) 2021 Université Gustave Eiffel
+#
+# Modification(s):
+#   - YYYY/MM Author: Description of the modification
+
+
+import numpy as np
+import scipy.spatial.distance as sc
+import warnings
+
+
+def _dist_to_diag(X, internal_p):
+    return ((X[:, 1] - X[:, 0]) * 2 ** (1. / internal_p - 1))
+
+
+def _build_dist_matrix(X, Y, order=2., internal_p=2):
```

Comment:
Could we move those 2 functions into a separate file (with a name that starts with `_` unless we want to document them) so we can avoid having several copies?

Reply:
I totally agree with this idea. How should it be done though? having a file `utils.py` in `wasserstein/` ?

Reply:
> I totally agree with this idea. How should it be done though? having a file `utils.py` in `wasserstein/` ?

If it is an internal file that users are not supposed to know about, I would say `_utils.py`, but yes.

Reply:
Done